pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -74,60 +76,115 @@
   }
 
   public static class ControllerPeriodicTasksConf {
+
     // frequency configs
+    @Deprecated
     public static final String RETENTION_MANAGER_FREQUENCY_IN_SECONDS = "controller.retention.frequencyInSeconds";
+    public static final String RETENTION_MANAGER_FREQUENCY_PERIOD = "controller.retention.frequencyPeriod";
     @Deprecated
     // The ValidationManager has been split up into 3 separate tasks, each having their own frequency config settings
     public static final String DEPRECATED_VALIDATION_MANAGER_FREQUENCY_IN_SECONDS =
         "controller.validation.frequencyInSeconds";
+    @Deprecated
```

Comment:
Another (separate) task can be to remove the `DEPRECATED_VALIDATION_MANAGER_FREQUENCY_IN_SECONDS` config and related code. I believe this has been deprecated for a few releases now, we can remove it.

Reply:
All deprecated configs have been prefixed with `DEPRECATED.` I will create a new ticket to do this in other places.

Reply:
I was suggesting to just remove `DEPRECATED_VALIDATION_MANAGER_FREQUENCY_IN_SECONDS`. It has been there since 0.5.0.  I understand this can mean some more code to remove. If that is too much, then you can choose to add a TODO there, saying that this config has been deprecated as of 0.5.0, so must be removed. Thanks

Reply:
Can you either remove `DEPRECATED_VALIDATION_MANAGER_FREQUENCY_IN_SECONDS` or add a comment before line 80 saying it has been deprecated since 0.5.0